src/test/java/api/item/ItemLastCheckInTests.java

```diff
@@ -71,7 +78,11 @@ public void shouldNotFailCheckInWithEmptyLoggedInUserId()
     JsonObject lastCheckIn = itemsClient.get(item.getId()).getJson()
       .getJsonObject("lastCheckIn");
 
-    assertThat(lastCheckIn.getString("dateTime"), is(checkInDate.toString()));
+    DateTime actualCheckinDateTime = DateTime.parse(lastCheckIn.getString("dateTime"));
+
+    assertThat((double) checkInDate.getMillis(),
+      is(closeTo((double) actualCheckinDateTime.getMillis(),
+        CURRENT_TIME_MARGIN_MILLIS)));
```

Comment:
Is the `TextDateTimeMatchers.withinSecondsAfter` matcher not appropriate for this use case?

Could the system time be frozen, so that this tests can check a specific date and time?

Reply:
1. TextDateTimeMatchers.withinSecondsAfter is not in this project.

Reply:
Done.

Reply:
Ok, given that the time can be frozen, can an exact comparison can be made?

(For reference, TextDateTimeMatchers.withinSecondsAfter is available in mod-circulation, it can be found [here](https://github.com/folio-org/mod-circulation/blob/3b15e6aa76fd14e97e6efec9f4a8d8f2217893d0/src/test/java/api/support/matchers/TextDateTimeMatcher.java#L59))

Reply:
There are exact comparisons in code, aren't there?

Reply:
Yes, there are exact comparisons in the tests now